### ISSUE TITLE
[Adaptive Cards-Blog page]: Under the Blog page, Blogs are not defined in List.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards#8883]
### Target Platforms

Other

### SDK Version

1.6

### Application Name

Adaptive Cards

### Problem Description

**Test environment:**
OS: Win11
OS Version: 24H2 (OS Build 26100.1)
URL:[Adaptive Cards](https://adaptivecards.io/)
Edge dev Browser version: 125.0.2506.0 (Official build) dev (64-bit)
Screen reader: Narrator.

**Repro Steps:** 
1. Open URL in the Edge browser & sign in with valid credentials. 
2. It will open the Adaptive cards home page 
3.Activate the blog control, navigate through tab key, and observe the issue. 

**Actual Result:**  
The cards in the Blog posts are navigable by the tab keys instead of up/down arrow keys and those cards are not defined in the list. 
 
**Expected Result:** 
The cards in the blog page should be defined in the list that should be navigable by the up/down arrow keys. 

**User Experience:**  
The keyboard users will be impacted if the Blog posts navigable by the tab key instead of arrow keys as if there are multiple posts the users will have to do multiple tabbing on the page to reach the desired location which will be confusing task. 

**WCAG Reference:** 
[MAS 2.1.1 Keyboard 
](https://www.w3.org/WAI/WCAG21/Understanding/keyboard)

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards/assets/95913748/26ac2dfc-7727-49a4-8dd7-dc6cf144bca6

![Blogs are not defined in list](https://github.com/microsoft/AdaptiveCards/assets/95913748/407be8c8-efaa-4159-bc9f-5e89f37200d7)

### Card JSON

```JSON
NA
```

**Comment by anna-dingler** [comment]
Hi @vagpt! I have a fix in progress for this right now. I enabled the left and right arrow keys so that users can navigate through the blog posts by going through each row. Let me know if this solution does not work for you - thanks!

**Comment by vagpt** [comment]
> Hi @vagpt! I have a fix in progress for this right now. I enabled the left and right arrow keys so that users can navigate through the blog posts by going through each row. Let me know if this solution does not work for you - thanks!
> 
>  Blog.post.keyboard.nav.mp4

Hi @anna-dingler 

For better user experience, Blogs Posts should be navigable with 4 arrow keys (Up, Down, Left and Right)

**Comment by vagpt** [comment]
Now this issue is working as expected, hence adding A11yTTValidated tag.

https://github.com/user-attachments/assets/d01bd96a-2e93-43fd-a14b-62c19b323777
